### PR TITLE
feat(api): bridge limits [SLT-165] 

### PR DESCRIPTION
**Pull request by bigboydiamonds**
PR introduces a new `/bridgeLimits` endpoint that provides the minimum and maximum origin input values for specific bridge routes. The endpoint allows users to query these limits by passing the following parameters:

	•	fromChain: The originating chain ID
	•	toChain: The destination chain ID
	•	fromToken: The address of the token on the originating chain
	•	toToken: The address of the token on the destination chain

Adds new `/bridgeLimits` Endpoint:
	•	The endpoint determines the min/max input values for a given bridge route.
	•	The input parameters (fromChain, toChain, fromToken, toToken) are used to match and return the appropriate route limits.
	•	returns { minOriginValue, maxOriginValue }

**Additional context**
Will be V1 initially deployed, following up with an optimization for resources.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new route for managing bridge limits between blockchain networks.
	- Added a controller to handle requests for bridge limits, including validation and response formatting.
	- New script command for generating limits and fetching bridge quotes.
	- Implemented functionality for constructing a JSON representation of swappable tokens across different blockchain networks.

- **Bug Fixes**
	- Improved address normalization for token lookups to enhance reliability.

- **Tests**
	- Added a test suite for the new bridge limits route to ensure correct functionality and error handling.

- **Documentation**
	- Updated package.json files to reflect new scripts and dependencies.

02d77f8b1c9eac0d64876674af18f4105e52f2bc: [synapse-interface preview link ](https://sanguine-synapse-interface-1hl20w0dq-synapsecns.vercel.app)

### CODE REVIEW

File: packages/rest-api/src/routes/bridgeLimitsRoute.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 error:
+ *                   type: string
+ *                 details:
+ *                   type: string
+ */
+router.get(
+  '/',
```
Comment (abtestingalpha) [inline packages/rest-api/src/routes/bridgeLimitsRoute.ts:97]: See other routes that have `normalizeNativeTokenAddress` and `checksumAddreses` middleware.

File: packages/rest-api/src/utils/tokenAddressToToken.ts [hunk 2]
```diff
@@ -1,19 +1,23 @@
+import { getAddress } from '@ethersproject/address'
+
 import { BRIDGE_MAP } from '../constants/bridgeMap'
 
 export const tokenAddressToToken = (chain: string, tokenAddress: string) => {
+  const address = getAddress(tokenAddress)
```
Comment (abtestingalpha) [inline packages/rest-api/src/utils/tokenAddressToToken.ts:6]: 1. Do we need to do this here? (note comment above where middleware will normalize & checksum addresses)

**Review by abtestingalpha — CHANGES_REQUESTED** [review]

File: packages/rest-api/src/routes/bridgeLimitsRoute.ts (hunk 1 above)
Comment (bigboydiamonds) [reply packages/rest-api/src/routes/bridgeLimitsRoute.ts:99]: Thanks, updated to use those middleware functions in https://github.com/synapsecns/sanguine/commit/30fd0c5fd08e6bfbc7ff9a2c71c09a87a9e4e634

File: packages/rest-api/src/utils/tokenAddressToToken.ts (hunk 2 above)
Comment (bigboydiamonds) [reply packages/rest-api/src/utils/tokenAddressToToken.ts]: Nope! Above middleware takes care of this, removed in https://github.com/synapsecns/sanguine/commit/30fd0c5fd08e6bfbc7ff9a2c71c09a87a9e4e634

**Review by abtestingalpha — APPROVED** [review]

**Pull request merged**